samples/nrf9160/lte_ble_gateway/boards/thingy91_nrf9160_ns.conf

```diff
@@ -0,0 +1,18 @@
+CONFIG_NRF_SW_LPUART=y
+CONFIG_NRF_SW_LPUART_INT_DRIVEN=y
+
+CONFIG_UART_1_ASYNC=y
+CONFIG_UART_1_INTERRUPT_DRIVEN=n
+CONFIG_UART_1_NRF_HW_ASYNC=y
+CONFIG_UART_1_NRF_HW_ASYNC_TIMER=2
+
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
```

Comment:
Didn't check this one, but based on other observation, could you please verify if this is needed ?

Reply:
Correct! Fixed.

Reply:
Fixed.

Reply:
The build fails when I try to remove it. It is enabled by something else, and it isn't prevented as SERIAL is enabled for the HCI communication to work.